src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -14,4 +21,41 @@ public String getRecoveredAdUnitId(String adUnitId) {
     jsActions.waitForJavaScriptTruthy(String.format("window._sp_ && window._sp_.getElementId && %s", getIdScript));
     return jsActions.execute(getIdScript).toString();
   }
+
+  public void verifyPageFairRecovery() {
+    By spansBodyChildrenSelector = By.cssSelector("body>span");
+    Dimension topLeaderboardSize = new Dimension(728, 90);
+    Dimension medrecSize = new Dimension(300, 250);
+
+    // when PF recovered ad is on page, inserts span elements as a direct children of body
+    wait.forElementPresent(spansBodyChildrenSelector);
+
+    // verify that adblock is turned on on that page
+    verifyNoAdsOnPage();
+
+    String firstSpanClass = driver.findElement(spansBodyChildrenSelector).getAttribute("class");
+    //System.out.println("\n\nCLASS:" + firstSpanClass);
+    List<WebElement> recoveredAds = driver
+        .findElements(By.cssSelector("body>span." + firstSpanClass))
+        .stream()
+        .filter(WebElement::isDisplayed)
+        .filter(e -> e.getCssValue("background").contains("data:image/jpeg"))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "background" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)